docs/get-started.md

```diff
@@ -0,0 +1,84 @@
+# Get Started with Kubeapps
```

Comment:
Can we avoid changing the filenames (i.e. getting-started -> get-started.md) if not critical, please? Easier to review and it won't break the links in kubeapps.com